assets/queries/ansible/aws/aws_config_rule_for_encrypted_vols_is_disabled/query.rego

```diff
@@ -0,0 +1,28 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  configRule := task["community.aws.aws_config_rule"]
+  configRuleName := task.name
+
+  upper(configRule.source.identifier) == "ENCRYPTED_VOLUMES"
```

Comment:
I think what this query should check is if there is no config rule identifying Encrypted Volumes as a source. 
There must at least one config rule identifying Encrypted Volumes as a source.

Reply:
You're right. I updated the query and the test samples

Reply:
The query will return a result for each "community.aws.aws_config_rule" with `configRule.source.identifier != "ENCRYPTED_VOLUMES"`. Try to get all tasks that are "community.aws.aws_config_rule" tasks and check if none of them has `configRule.source.identifier == "ENCRYPTED_VOLUMES"`. That way, the query will return only one result. I think the searchKey could point to the first "community.aws.aws_config_rule" task and, also, the issueType should be "MissingAttribute". Check the Terraform's version of this query if you have any questions

Reply:
I think it's ok now. Thanks for the explanation :)